src/views/v2/day/event/featured-image.php

```diff
@@ -22,12 +22,18 @@
 ?>
 <div class="tribe-events-calendar-day__event-featured-image-wrapper tribe-common-g-col">
 	<div class="tribe-events-calendar-day__event-featured-image tribe-common-c-image tribe-common-c-image--bg">
-		<div
-			class="tribe-common-c-image__bg"
-			style="background-image: url('<?php echo get_the_post_thumbnail_url( $event_id, 'large' ); ?>');"
-			role="img"
-			aria-label="alt text here"
+		<a
+			href="#"
+			title="<?php echo esc_attr( get_the_title( $event_id ) ); ?>"
+			rel="bookmark"
 		>
-		</div>
+			<div
+				class="tribe-common-c-image__bg"
+				style="background-image: url('<?php echo get_the_post_thumbnail_url( $event_id, 'large' ); ?>');"
```

Comment:
:warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 38986e95dfa4f92db853d5d7f0039d7d</sub>